lib/streamlit/elements/json.py

```diff
@@ -70,6 +70,9 @@ def json(
 
         if not isinstance(body, str):
             try:
+                for key in body:
+                    if type(body[key]) == set:
```

Comment:
This might be slightly better expressed as `if isinstance(body[key], set)` - this will also handle `set` subclasses

Reply:
Thank you `isinstance` is much better here, I've introduced this change into the PR 👍